src/automations/components/AutomationActionDescriptionSendNotification.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="automation-action-description-send-notification" v-bind="$attrs">
+    Send {{ indefiniteArticle() }} {{ blockDocument?.blockType.name.toLocaleLowerCase() ?? '' }} notification using
+    <BlockIconText :block-document-id="blockDocumentId" />
+    {{ recipients }}
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { computed } from 'vue'
+  import { AutomationActionSendNotification } from '@/automations/types/actions'
+  import { BlockIconText } from '@/components'
+  import { useBlockDocument } from '@/compositions'
+
+  defineOptions({
+    inheritAttrs: false,
+  })
+
+  const props = defineProps<{
+    action: AutomationActionSendNotification,
+  }>()
+
+  const { blockDocument } = useBlockDocument(props.action.blockDocumentId)
```

Comment:
Accessing `props.action.blockDocumentId` directly like this will break the reactivity because it's passing the value of the id but it cannot track the effects. Here you can use a getter

```suggestion
  const { blockDocument } = useBlockDocument(() => props.action.blockDocumentId)
```